decidim-core/lib/decidim/core/test/shared_examples/uncommentable_component_examples.rb

```diff
@@ -18,4 +18,20 @@
       expect(page).to have_no_link(resource_locator(resource).path)
     end
   end
+
+  describe "when search a comment in the global search" do
+    let(:comment) { create(:comment) }
+
+    it "does not displays the comment" do
```

Comment:
Could you test that the result is actually findable before disabling the comments?

Reply:
@alecslupu done!